src/Chunk.ts

```diff
@@ -133,6 +136,22 @@ export default class Chunk {
 		this.name = makeLegal(id);
 	}
 
+	generateMangledExportNames () {
+		let curIndex = 9;
+		this.mangledExportNameMap = {};
+		Object.keys(this.exports).forEach(exportName => {
+			if (exportName[0] === '*')
+				return;
+			let mangledName: string;
+			do {
+				curIndex++;
+				mangledName = curIndex.toString(36);
+			}
+			while (mangledName.charCodeAt(0) > 47 && mangledName.charCodeAt(0) < 58)
```

Comment:
Does not feel like the most straight-forward way of doing this but I must admit I cannot come up with anything that has better performance.